bot/cogs/watchchannels/watchchannel.py

```diff
@@ -0,0 +1,353 @@
+
+            message = textwrap.dedent(
+                f"""
+                An error occurred while loading the text channel or webhook.
+
+                TextChannel: {"**Failed to load**" if self.channel is None else "Loaded successfully"}
+                Webhook: {"**Failed to load**" if self.webhook is None else "Loaded successfully"}
+
+                The Cog has been unloaded.
+                """
+            )
+
+            await self.modlog.send_log_message(
+                title=f"Error: Failed to initialize the {self.__class__.__name__} watch channel",
+                text=message,
+                ping_everyone=True,
+                icon_url=Icons.token_removed,
+                colour=Color.red()
+            )
+
+            self.bot.remove_cog(self.__class__.__name__)
+            return
+
+        if not await self.fetch_user_cache():
+            await self.modlog.send_log_message(
+                title=f"Warning: Failed to retrieve user cache for the {self.__class__.__name__} watch channel",
+                text="Could not retrieve the list of watched users from the API and messages will not be relayed.",
+                ping_everyone=True,
+                icon=Icons.token_removed,
+                color=Color.red()
+            )
+
+    async def fetch_user_cache(self) -> bool:
+        """
+        Fetches watched users from the API and updates the watched user cache accordingly.
+
+        This function returns `True` if the update succeeded.
+        """
+        try:
+            data = await self.bot.api_client.get(self.api_endpoint, params=self.api_default_params)
+        except aiohttp.ClientResponseError as e:
+            self.log.exception(f"Failed to fetch the watched users from the API", exc_info=e)
+            return False
+
+        self.watched_users = defaultdict(dict)
+
+        for entry in data:
+            user_id = entry.pop('user')
+            self.watched_users[user_id] = entry
+
+        return True
+
+    async def on_message(self, msg: Message) -> None:
+        """Queues up messages sent by watched users."""
+        if msg.author.id in self.watched_users:
+            if not self.consuming_messages:
+                self._consume_task = self.bot.loop.create_task(self.consume_messages())
+
+            self.log.trace(f"Received message: {msg.content} ({len(msg.attachments)} attachments)")
+            self.message_queue[msg.author.id][msg.channel.id].append(msg)
+
+    async def consume_messages(self, delay_consumption: bool = True) -> None:
+        """Consumes the message queues to log watched users' messages."""
+        if delay_consumption:
+            self.log.trace(f"Sleeping {BigBrotherConfig.log_delay} seconds before consuming message queue")
+            await asyncio.sleep(BigBrotherConfig.log_delay)
+
+        self.log.trace(f"{self.__class__.__name__} started consuming the message queue")
```

Comment:
Remove the name from the message.

```suggestion
        self.log.trace(f"Started consuming the message queue")
```